### PR TITLE
Sdkman Installation

### PR DESCRIPTION
**Pull request by kamleshkc2002**
Adding SDKMan to the repository

Relevant Issue:
https://github.com/sb2nov/mac-setup/issues/201#issue-318911082

### CODE REVIEW

File: Java/sdkman.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# `SdkMan`
```
Comment (hugovk) [inline Java/sdkman.md:1]:
```suggestion
# SDKMAN
```

File: Java/sdkman.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+SDKMAN! is a tool for managing parallel versions of multiple Software Development Kits on most Unix based systems. It provides a convenient Command Line Interface (CLI) and API for installing, switching, removing and listing Candidates.
```
Comment (hugovk) [inline Java/sdkman.md:3]:
```suggestion
SDKMAN! is a tool for managing parallel versions of multiple software development kits on most Unix-based systems. It provides a convenient command line interface (CLI) and API for installing, switching, removing and listing candidates.
```

File: Java/sdkman.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## Installation
+
+Simply open a new terminal and enter:
```
Comment (hugovk) [inline Java/sdkman.md:7]:
```suggestion
Open a new terminal and enter:
```

File: Java/sdkman.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+```bash
+sdkman 5.0.0+51
+```
+
+## Usage
+
+### Installing an SDK
+
+#### Latest Stable
```
Comment (hugovk) [inline Java/sdkman.md:36]:
```suggestion
#### Latest stable
```

File: Java/sdkman.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+```bash
+Do you want java 8u111 to be set as default? (Y/n):
+Answering yes (or hitting enter) will ensure that all subsequent shells opened will have this version of the SDK in use by default.
+Setting java 8u111 as default.
+```
+
+## More commands
+
+* To install a specific version
+simply qualify the version you require
```
Comment (hugovk) [inline Java/sdkman.md:68]:
```suggestion
qualify the version you require:
```

File: Java/sdkman.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+* To remove a version
+
+```bash
+sdk uninstall scala 2.11.6
+```
+
+Note that removing a local version will not remove the local installation.
+
+* List Candidates
```
Comment (hugovk) [inline Java/sdkman.md:82]:
```suggestion
* To list candidates
```

File: Java/sdkman.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+```bash
+sdk list
+```
+
+More useful commands and usages can be found at
+
+[SDKMAN Usage](https://sdkman.io/usage)
```
Comment (hugovk) [inline Java/sdkman.md:90]:
```suggestion
More useful commands and usages can be found at [SDKMAN Usage](https://sdkman.io/usage).
```

**Comment by kamleshkc2002** [comment]
Added all the suggestions from @hugovk

**Pull request merged**